### PR TITLE
fix model.py compatibility with non-id-gnn and set process=False for fast Database loading

### PR DESCRIPTION
**Pull request by rishabh-ranjan**
.

### CODE REVIEW

File: examples/model.py [hunk 1]
```diff
@@ -75,15 +75,14 @@ def reset_parameters(self):
         self.head.reset_parameters()
         for embedding in self.embedding_dict.values():
             torch.nn.init.normal_(embedding.weight, std=0.1)
-        self.id_awareness_emb.reset_parameters()
+        if self.id_awareness_emb is not None:
+            self.id_awareness_emb.reset_parameters()
```
Comment (weihua916) [inline examples/model.py:79]: Good catch, thanks

File: examples/gnn_link.py [hunk 2]
```diff
@@ -50,7 +50,7 @@
 root_dir = "./data"
 
 # TODO: remove process=True once correct data/task is uploaded.
-dataset: RelBenchDataset = get_dataset(name=args.dataset, process=True)
+dataset: RelBenchDataset = get_dataset(name=args.dataset, process=False)
```
Comment (weihua916) [inline examples/gnn_link.py:53]: Does this work for all datasets? For example, I believe HM is not working

**Review by weihua916 — APPROVED** [review]
Thanks, although I believe `process=True` should be retrained until we upload all datasets to our server

File: examples/gnn_link.py (hunk 2 above)
Comment (rishabh-ranjan) [reply examples/gnn_link.py:53]: ah yes. i will upload the other databases too (assuming that the databases are stable even though we are still iterating on the task definitions).

currently database is processed every time the script runs and wastes a lot of time in startup.

Comment (weihua916) [reply examples/gnn_link.py:53]: Right. How about leaving it as `True` until our final release? I usually find it ok (~2min to start). Feel free to make it `False` locally.

Comment (rishabh-ranjan) [reply examples/gnn_link.py:53]: I have uploaded the other databases. So process=False works correctly. But I will revert to process=True for the public code as you say.

Comment (weihua916) [reply examples/gnn_link.py]: Thanks!

**Pull request merged**